pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -814,3 +837,14 @@ func updateAllocationPerNUMAMetric(topo *topology.CPUTopology, allocatedCPUs cpu
 		metrics.CPUManagerAllocationPerNUMA.WithLabelValues(strconv.Itoa(numaNode)).Set(float64(count))
 	}
 }
+
+func (p *staticPolicy) isPodWithPodLevelResources(pod *v1.Pod) bool {
+	if utilfeature.DefaultFeatureGate.Enabled(features.PodLevelResources) && resourcehelper.IsPodLevelResourcesSet(pod) {
+		// The Memory manager static policy does not support pod-level resources.
```

Comment:
stale comment

Reply:
resolved in follow up PR - https://github.com/kubernetes/kubernetes/pull/133256